src/styles/isomer-cms/generic/base.scss

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700,900&display=swap');
+
+body{
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  background: $base-background;
+  margin: 0;
+  padding: 0;
+  color: $body-text;
+}
+
+a{
+  text-decoration: none;
+  color: $isomer-blue;
+  padding: 0;
+  margin: 0;
+
+  &:hover{
+    text-decoration: none;
+    color: $isomer-blue;
+  }
+}
+
+.wrapper{
+  height: calc(100vh - #{$header-height});
+  width: 100%;
+  top: $header-height;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+
+}
+
+h1,h2,h3,h4,h5,h6{
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  margin: 0;
+  padding: 0;
+}
+
+
+.page-editor-footer{
+  height: $footer-height;
+  background: white;
+  box-shadow: 0 1px 6px 0 rgba(0, 0, 0, 0.5);
+  position: absolute;
+  bottom: 0;
+  z-index: 10;
```

Comment:
Use z-index sparingly and the z-index value should be set as a CSS variable to prevent accidental "stacking wars"